qkernel/makefile

```diff
@@ -43,16 +43,16 @@ $(cc_kernel_debug): cc_kernel_debug $(assembly_object_files)
 
 
 kernel:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release
+	CARGO_TARGET_DIR=../target cargo +nightly-2023-12-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release
 
 kernel_debug:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json
+	CARGO_TARGET_DIR=../target cargo +nightly-2023-12-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json
 
 cc_kernel:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release --features cc
+	CARGO_TARGET_DIR=../target cargo +nightly-2023-12-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release --features cc
 
 cc_kernel_debug:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --features cc
```

Comment:
please drop this, as it's already merged into main. (It might bite you XD)

Reply:
Thank you, will do it.